CourseApp.Tests/TasksTest.cs

```diff
@@ -0,0 +1,84 @@
+namespace CourseApp.Tests
+{
+    using Xunit;
+
+    public class TasksTest
+    {
+        [Theory]
+        [InlineData(-2.17041, -1.77066, -1.47417, -1.23865, -1.04481, -0.88217)]
```

Comment:
if you trust in you test for calculation - you could simply check length of returned array